youtube/generic.md

```diff
@@ -1,50 +1,71 @@
-...
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
```

Comment:
Hmm, shouldn't this be `'` instead of `’`?
```suggestion
Don't forget to support my channel by subscribing to it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
```